ts/src/profiler.ts

```diff
@@ -28,6 +28,7 @@ import {ProfilerConfig} from './config';
 import {logLevelToName} from './index';
 import * as heapProfiler from './profilers/heap-profiler';
 import {TimeProfiler} from './profilers/time-profiler';
+import {create, SourceMapper} from './sourcemapper/sourcemapper';
```

Comment:
It would be self-documenting to do `import {create as createSourceMapper, SourceMap} ...`.

Reply:
Done.